lib/common.sh

```diff
@@ -95,7 +95,7 @@ FILESYSTEM=${FILESYSTEM:="/"}
 
 export CAPI_VERSION="${CAPI_VERSION:-"v1alpha3"}"
 export CAPM3_VERSION="${CAPM3_VERSION:-"v1alpha4"}"
```

Comment:
This CAPI and CAPM3 defaults should probably be changed, but maybe later, once v1a4 is properly on the way

Reply:
Yes, lets do this when v1a4 lands